src/oop_ext/interface/_interface.py

```diff
@@ -603,15 +665,19 @@ def _IsMethod(member):
 
 
 @Deprecated(AssertImplements)
-def AssertImplementsFullChecking(class_or_instance, interface, check_attr=True):
```

Comment:
Looks like `@Deprecated` is not correctly typed
```
_interface.py:668: error: Function is untyped after decorator transformation

Reply:
Again not seeing any errors either on CI or locally...